common/src/main/java/com/microsoft/identity/common/internal/authorities/AADTestAuthority.java

```diff
@@ -0,0 +1,59 @@
+package com.microsoft.identity.common.internal.authorities;
```

Comment:
Add copyright of all the new classes and these should be part of the test directory

Reply:
Addressed.

Moved all test classes to a separate module that is solely for test purposes.